packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -622,6 +697,33 @@ class VectorGraphicsCodec {
     final double height = buffer.getFloat32();
     listener?.onSize(width, height);
   }
+
+  void _readTextConfig(
+    _ReadBuffer buffer,
+    VectorGraphicsCodecListener? listener,
+  ) {
+    final int id = buffer.getUint16();
+    final double dx = buffer.getFloat32();
+    final double dy = buffer.getFloat32();
+    final double fontSize = buffer.getFloat32();
+    final int fontWeight = buffer.getUint8();
+    String? fontFamily;
+    final int fontFamilyLength = buffer.getUint16();
+    if (fontFamilyLength > 0) {
+      fontFamily = utf8.decode(buffer.getUint8List(fontFamilyLength));
+    }
+    final int textLength = buffer.getUint16();
+    final String text = utf8.decode(buffer.getUint8List(textLength));
```

Comment:
We should either assert that textLength > 0 or handle when it is zero.

Reply:
Added test that shows that this works fine. Similar to the other methods here, it isn't the responsibility of the codec to optimize so we should allow it

Reply:
In other methods, if the uint8list is zero length we do not encode it or decode it though...

Reply:
I do that because they're nullable. Encoding a 0 length list is free